mongo/dataset_store.go

```diff
@@ -299,7 +300,7 @@ func (m *Mongo) UpdateDataset(id string, dataset *models.Dataset) (err error) {
 	defer s.Close()
 
 	updates := createDatasetUpdateQuery(dataset)
-
+	log.Debug("Stuff", log.Data{"id": id, "updates": updates})
```

Comment:
remove plz (or change 'Stuff' to a more meaningful message) - and remove the import!